projects/packages/my-jetpack/changelog/remove-red-bubble-data-from-window-state

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Take red bubble notifications out of initial state
```

Comment:
Now that I prepare changelogs for releases - our best practice is to write a module name upfront, and end the changelog always with the coma. For example:
`My Jetpack: Move Red Bubble notifications out of myJetpackInitialState.`

It's a small suggestion, though, not related to the PR code itself!